sources/_sector/dex/trades/blast/_sources.yml

```diff
@@ -17,3 +17,8 @@ sources:
       - name: ThrusterFactory_Point_3_Fee_evt_PairCreated
       - name: ThrusterPair_V2_1_Fee_evt_Swap
       - name: ThrusterFactory_1_Fee_evt_PairCreated
+  - name: bladeswap_blast
+    tables:
+      - name: XYKPoolFactory_V2_evt_PoolCreated
+      - name: XYKPool_evt_Swap
+      - name: Vault_Router_evt_Swap
```

Comment:
The swap events here may not be very clear:
https://dune.com/queries/3925455

here is an example event log:
https://blastscan.io/tx/0x3f9a53ffe1369136af86c1fff61868ede338be17ace446b8880249c371379d10#eventlog

Reply:
Sample bladeswap_blast.base_trades based on manually decoding:
https://dune.com/queries/3925627

Reply:
However multiple tokens (>2) swapping may happen here.